### DISCUSSION TITLE
ATMEGA2560 AVR?

### DISCUSSION THREAD

**Discussion by borodiliz** [IOTEROP/IOWA#3] [General] [answered]
Hi!,

We're evaluating the use of LwM2M in our products. Is it possible to use your IOWA Full SDK on an ATMEGA2560 AVR? If so, what are the RAM/ROM requirements? 

Regards

**Comment by oguiter** [comment, marked as answer]
Hi @borodiliz,
The IOWA stack can run on ATMEGA2560 but the limited memory of the chip could lead you to make some "wizardry". Moreover, you must think of a way to communicate externally, and so, add some code.
At Ioterop, we recently did a port on an AVR128DA, which is much more flexible from our point of view (and cheaper !) (with udp over slip).
(I will forward your email to the person in charge of the IOWA Full SDK package)

**Comment by oguiter** [comment]
Hi  @borodiliz,
Could you please send an email to [contact@ioterop.com](contact@ioterop.com) ? We can discuss about the IOWA Full SDK ..
